projectview/src/main/java/org/jetbrains/bsp/bazel/projectview/parser/splitter/ProjectViewRawSections.java

```diff
@@ -0,0 +1,44 @@
+package org.jetbrains.bsp.bazel.projectview.parser.splitter;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import org.apache.commons.collections4.CollectionUtils;
+
+public class ProjectViewRawSections {
+
+  private final List<ProjectViewRawSection> sections;
+
+  public ProjectViewRawSections(List<ProjectViewRawSection> sections) {
+    this.sections = sections;
+  }
+
+  public Optional<ProjectViewRawSection> getLastSectionWithName(String sectionName) {
+    return getAllWithName(sectionName).stream().reduce((first, second) -> second);
+  }
+
+  public List<ProjectViewRawSection> getAllWithName(String sectionName) {
```

Comment:
Maybe return a stream here, since that is what you need in all the non-test consumers of this method anyway.

Reply:
yea, it think that I should avoid those unnecessary `.collect(Collectors.toList())` cuz I do `.stream()` later anyway